locales/en/app.ftl

```diff
@@ -55,9 +57,8 @@ user-unsubscribed-title = {-product-name} : Unsubscribed
 pwt-section-headline = Stronger Passwords = Better Protection
 pwt-section-subhead = Your private information is only as safe as your passwords.
 pwt-section-blurb =
-  Your passwords protect more than your accounts. They protect every bit of personal information that resides in them. 
-  And hackers rely on bad habits, like using the same password everywhere or using common phrases (p@ssw0rd, anyone?) so 
-  that if they hack one account, they can hack many. Here’s how to better protect your accounts.
+  Passwords protect all the personal information in your online accounts. 
+  And hackers rely on bad habits, like using the same password everywhere or using common phrases (@p@ssw0rd, anyone?) so that if they hack one account, they can hack many.
```

Comment:
All these string changes require new IDs.
https://developer.mozilla.org/en-US/docs/Making_String_Changes

I don't think the new lines are meaningful here? If they're not, I'd try a more consistent approach to multiline text.

```
SOMENEWID = 
    Passwords protect all the personal information in your online accounts. And
    hackers rely on bad habits, like using the same password everywhere or using
    common phrases (@p@ssw0rd, anyone?) so that if they hack one account, they
    can hack many.
```